packages/check/src/main.rs

```diff
@@ -11,7 +11,7 @@ use cosmwasm_vm::capabilities_from_csv;
 use cosmwasm_vm::internals::{check_wasm, compile, make_compiling_engine};
 
 const DEFAULT_AVAILABLE_CAPABILITIES: &str =
-    "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3";
+    "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4";
```

Comment:
This change is independent of the PR, right? We don't need a capability for this change. See "What's a good capability?" in https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES.md#whats-a-good-capability

Reply:
Yes, good point! I originally thought we would need it to ensure that a contract with the imports can only be uploaded to 1.4 chains, but that will just fail on older chains because of the unsupported imports, so we don't need the capability.